usaspending_api/search/v2/views/spending_by_category_views/spending_by_category.py

```diff
@@ -56,6 +52,15 @@ class AbstractSpendingByCategoryViewSet(APIView, metaclass=ABCMeta):
     pagination: Pagination
     high_cardinality_categories: List[str] = ["recipient", "recipient_duns"]
     spending_level: Optional[SpendingLevel]
+    subaward_agg_key_mapper = {
```

Comment:
Could add a type hint to be consistent with the above
```suggestion
    subaward_agg_key_mapper: dict[str, str] = {
```

Reply:
Added the type hint.